apps/frontend/src/libs/helpers/get-user-columns/get-user-columns.ts

```diff
@@ -0,0 +1,20 @@
+import { type ColumnDef } from "@tanstack/react-table";
```

Comment:
Re-export this type as TableColumn in common types, then use it in Table component too. We don't want to use library import everywhere